### ISSUE TITLE
[css-subtree-visibility-1] should the word 'subtree' be exposed as part of a CSS property name

### ISSUE THREAD

**Issue by dbaron** [WICG/display-locking#127]
@kenchris and I are looking at [the CSS Subtree Visibility draft](https://wicg.github.io/display-locking/) as part of w3ctag/design-reviews#306.

We're not aware of other CSS properties exposing the word "subtree", and it's not clear what terms we should be using here.  While as English speakers we may interchangeably use "subtree" and "descendants" (although the former probably includes the element itself and the latter does not), exposing additional technical terms to the set of English words that we're exposing to authors should probably be considered a little more carefully, particularly in this somewhat sensitive area of tree terminology where we already have some distinctions that people are often confused by (e.g., elements versus descendants).

**Comment by vmpstr** [comment]
> although the former probably includes the element itself and the latter does not

Subtree in the way that we've been using it does not include the element itself either. 

Do you have a recommendation for a word that's more appropriate here? To your point in #128 it's probably worthwhile to have terms that unambiguously refer to a particular tree. 'Descendant' when talking about css combinators is I think talking about a the DOM tree? Or perhaps it's referring to a descendant in any tree, since I can't really think of a case where the descendant in the DOM would not be a descendant in a flat tree.

```
.e {
  descedant-visibility: ...
}
```
to me seems like it could be confused with
```
.e * {
  visibility: ...
}
```
In other words, "descendant visibility of class e" vs "visibility of descendants of class e"

**Comment by fantasai** [comment]
I suggest `content-visibility` since `content` is a term we expose already to refer to an element's contents.

**Comment by vmpstr** [comment]
This was renamed to `content-visibility` in https://github.com/WICG/display-locking/pull/155
